javascript/javascript1/week1/hw1.js

```diff
@@ -0,0 +1,54 @@
+// A future age calculator 
+
+const yearOfBirth = 1987;
+const yearOfFuture = 2027;
+const age = yearOfFuture - yearOfBirth;
+ const output = ` you will be ${age} years old in ${yearOfFuture}`;
+ console.log(output);
+
+
+// A dog age calculator
+
+ const dogYearOfBirth = 2017;
+ const dogYearFuture = 2027;
+ const dogYear = dogYearFuture - dogYearOfBirth;
+ var shouldShowResultInDogYears; 
+ if(shouldShowResultInDogYears == dogYear * 7)
+ {
+     shouldShowResultInDogYears = `your dog will be ${shouldShowResultInDogYears} dog yeras old in ${dogYearFuture}`;   
+ }else {
+    shouldShowResultInDogYears = `your dog will be ${dogYear} human yeras old in ${dogYearFuture}`;
```

Comment:
Same thing as above, you will want to `console.log` this, not assign it to a variable.